gio/Gir.toml

```diff
@@ -990,6 +990,20 @@ status = "generate"
     #readonly
     ignore = true
 
+[[object]]
+name = "Gio.SimpleProxyResolver"
+status = "generate"
+manual_traits = ["SimpleProxyResolverExtManual"]
+    [[object.function]]
+    name = "new"
+    # takes vec of strings
+    manual = true
```

Comment:
What is autogenerated here?

Reply:
it generates a plain `&str`... not sure why, the annotation for array seems to be there in the C code